tests/convert-3d-view-to-svg.test.ts

```diff
@@ -0,0 +1,40 @@
+import type { AnySoupElement } from "@tscircuit/soup"
+import { expect, test } from "bun:test"
+import { convert3dCircuitToSvg } from "../src/three-components/convert-3d-circuit-to-svg.tsx"
+import circuitJson from "./assets/circuit.json"
+import { applyJsdomShim } from "src/utils/jsdom-shim.ts"
+import { JSDOM } from "jsdom"
+
+test("convert 3d view to svg of a single resistor component", async () => {
+  const dom = new JSDOM()
+  applyJsdomShim(dom)
+
+  const options = {
+    width: 800,
+    height: 600,
+    backgroundColor: "#ffffff",
+    padding: 20,
+    zoom: 50,
+    viewAngle: "isometric" as const,
+    camera: {
+      position: {
+        x: 50,
+        y: 50,
+        z: 100,
+      },
+      lookAt: {
+        x: 0,
+        y: 0,
+        z: 0,
+      },
+    },
+  }
+
+  const svgString = await convert3dCircuitToSvg(
+    circuitJson as AnySoupElement[],
+    dom,
```

Comment:
why is dom a parameter? If dom is a parameter then you don't need the shim right?